conanfile.py

```diff
@@ -32,9 +33,14 @@ class libsocket(ConanFile):
     source_subfolder = "source_subfolder"
     build_subfolder = "build_subfolder"
 
+
     def source(self):
         self.run("git clone https://github.com/dermesser/libsocket --depth 1 %s" % self.source_subfolder)
 
+        # patching CMakeLists.txt in Source Directory
```

Comment:
are patches really applied?
if not, please remove commented code and patch file